packages/headless/src/features/generated-answer/generated-answer-analytics-actions.ts

```diff
@@ -17,8 +17,26 @@ export type GeneratedAnswerFeedback =
   | 'notAccurate'
   | 'outOfDate'
   | 'harmful';
+
+export type GeneratedAnswerFeedbackOption = 'yes' | 'unknown' | 'no';
+
+export type GeneratedAnswerFeedbackV2 = {
```

Comment:
Will we rename this type to become `GeneratedAnswerFeedback` starting from the v3?
In that case we need another `TODO` comment